openvdb/openvdb/unittest/TestTools.cc

```diff
@@ -1600,21 +1600,22 @@ struct FloatToVec
     // Transform a scalar value into a vector value.
     static openvdb::Vec3s toVec(const ValueT& v) { return openvdb::Vec3s(v, v*2, v*3); }
 
-    FloatToVec() { numTiles = 0; }
+    FloatToVec() : numTiles{0} {}
+    FloatToVec(const FloatToVec& other) : numTiles{other.numTiles.load(std::memory_order_acquire)} {}
 
     void operator()(const InIterT& it, Accessor& acc)
     {
         if (it.isVoxelValue()) { // set a single voxel
             acc.setValue(it.getCoord(), toVec(*it));
         } else { // fill an entire tile
-            numTiles.fetch_and_increment();
```

Comment:
You would think with ambiguous api calls like this it would be
fetch_then_increment
and
fetch_then_add
to make it clear that the result of the fetch is from before the operation in both cases.   The fetch_add official docs do not help with wonderful lines like

Return value
The value immediately preceding the effects of this function in the modification order of *this.

That was helpful!